src/darwin/Framework/CHIP/templates/partials/attribute_data_callback_name.zapt

```diff
@@ -14,8 +14,22 @@
       VendorId
     {{~else if_is_strongly_typed_chip_enum type~}}
         {{asUpperCamelCase parent.name preserveAcronyms=true}}Cluster{{asUpperCamelCase type preserveAcronyms=true}}
+    {{~else if (isOctetString type)~}}
+      OctetString
+    {{~else if (isCharString type)~}}
+    CharString
+    {{~else if (isArray)~}}
+    List
```

Comment:
This cannot possibly happen; isArray is the very first thing tested for above and does something _quite_ different.

Reply:
ok I may have picked this up from the js code. I can remove it if that is the case